phraseapp/config.go

```diff
@@ -157,6 +157,28 @@ func ValidateIsRawMap(k string, v interface{}) (map[string]interface{}, error) {
 	return ps, nil
 }
 
+func ValidateIsRawStringMap(k string, v interface{}) (map[string]string, error) {
+	raw, err := ValidateIsRawMap(k, v)
```

Comment:
The naming is off. You do not just validate the map, you validate it and typecast it, returning a map[string]string, maybe do the validateIsRawMap before or (because of ego templating) rename it to reflect that a string is cast to a map.